discovery/src/test/java/resources/apollo.properties

```diff
@@ -0,0 +1,5 @@
+#apollo config
```

Comment:
Is it necessary?

Reply:
Non-essential, as a configuration reference example in apollo, similar to nacos-config.txt